flixel/FlxState.hx

```diff
@@ -55,44 +55,23 @@ class FlxState extends FlxContainer
 	var _constructor:NextState;
 	
 	/**
-	 * Current substate. Substates also can be nested.
+	 * Current substate. `FlxSubState`s can also be nested.
 	 */
 	public var subState(default, null):FlxSubState;
 
-	/**
-	 * If a state change was requested, the new state object is stored here until we switch to it.
-	 */
-	@:noCompletion
-	var _requestedSubState:FlxSubState;
-
-	/**
-	 * Whether to reset the substate (when it changes, or when it's closed).
-	 */
-	@:noCompletion
-	var _requestSubStateReset:Bool = false;
-
 	/**
 	 * A `FlxSignal` that dispatches when a sub state is opened from this state.
 	 * @since 4.9.0
 	 */
-	public var subStateOpened(get, never):FlxTypedSignal<FlxSubState->Void>;
+	public var subStateOpened(default, null):FlxTypedSignal<FlxSubState->Void> = new FlxTypedSignal<FlxSubState->Void>();
 
 	/**
 	 * A `FlxSignal` that dispatches when a sub state is closed from this state.
 	 * @since 4.9.0
 	 */
-	public var subStateClosed(get, never):FlxTypedSignal<FlxSubState->Void>;
-
-	/**
-	 * Internal variables for lazily creating `subStateOpened` and `subStateClosed` signals when needed.
-	 */
-	@:noCompletion
-	var _subStateOpened:FlxTypedSignal<FlxSubState->Void>;
-
-	@:noCompletion
-	var _subStateClosed:FlxTypedSignal<FlxSubState->Void>;
+	public var subStateClosed(default, null):FlxTypedSignal<FlxSubState->Void> = new FlxTypedSignal<FlxSubState->Void>();
 	
-	public function new ()
+	public function new():Void
```

Comment:
what? why?!

Reply:
This is just a habit i guess, I didn't even realize i made this change until now